metric_descriptions/catalog-coverage.yml

```diff
@@ -0,0 +1,31 @@
+name: Catalog Coverage
+
+summary: > 
+    The percentage (%) of the division of the unique services found in recommendations to the total number of published services
+
+description: > 
+   The Catalog Coverage is described by the formula $$\frac{unique rec_services}{services}$$
+
+output:
+    type: float
+    min: 0
+    max: 100
+    comment: Catalog Coverage is 0 when none of the services is being recommended, and 100 when all of them are being recommended.
+
+prerequisites:
+    - all available recommendations
+    - all available services
+
+process:
+    - step: Retrieve recommendations
```

Comment:
all recommendations? with anonymous and registered users?

Reply:
yes, both of them